File: JH/DAY5/1924.cpp
#include <bits/stdc++.h>

using namespace std;


int a,b;

int main(){

    cin >> a >> b;

    int day = 0;

    for (int i = 1; i <a ; ++i) {
        if(i==1 || i==3 || i==5 || i==7 || i==8 || i==10 || i==12 ){
            day+=31;
        }else if(i==4||i==6||i==9||i==11){
            day+=30;
        }else if(i==2){
            day+=28;
        }
    }
    day+=b;
    int c = day % 7;
    if(c == 1){
        cout << "MON";
    }
    if(c == 2){
        cout << "TUE";
    }
    if(c == 3){
        cout << "WED";
    }
    if(c == 4){
        cout << "THU";
    }
    if(c == 5){
        cout << "FRI";
    }
    if(c == 6){
        cout << "SAT";
    }
    if(c == 0){
        cout << "SUN";
    }

}

File: HY/Day 4/1924.py
a = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"]  # 요일을 담은 list a
# 각 월별 총 일수를 담은 list b[0]은 1월 b[1]은 2월 ..... b[11]은 12월의 총 일수이다
b = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
day = 0

m, d = map(int, input().split())  # 입력값 받기 m-월 d-일

for i in range(0, m-1):  # 입력한 월 전까지의 총 일수를 다 더하기 예를 들어, 1월이면 아무것도 더하지 않고, 3월이면 1월~2월까지의 일수를 더한다
    day += b[i]


answer = (day + d) % 7  # 요일은 day + 입력 일수 중 7로 나눈 나머지이다.

print(a[answer])
